flow/alerting/classifier.go

```diff
@@ -468,6 +471,10 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 			return ErrorOther, myErrorInfo
 		case 1146: // ER_NO_SUCH_TABLE
 			return ErrorNotifySourceTableMissing, myErrorInfo
+		case 1943:
```

Comment:
https://mariadb.com/docs/server/reference/error-codes/mariadb-error-codes-1900-to-1999/e1943

No need to check for HY000 & error message